### ISSUE TITLE
jQuery UI Accordion

### ISSUE THREAD

**Issue by jteutenberg2** [ehynds/jquery-ui-multiselect-widget#702]
List of entries isn't shown correctly on top in a jQuery UI accordion area.

**Comment by mlh758** [comment]
Are you using the 2.x code and if so do you have a fiddle to demonstrate
the issue?

**Comment by jteutenberg2** [comment]
I am using 2.0.1.
I don’t have a fiddle to demonstrate it.
The generated list box isn’t shown in the foreground.
It is covered by the title of the accordion.

[cid:image004.png@01D2F55E.26E94B40]

Same problem on jQuery UI tabs.
It isn’t possible to use the multiselect widget on a tab.
The list is covered by the content of the tab.
[cid:image005.png@01D2F55E.26E94B40]

The problem exists with different browsers.

Outside the accordion and tab it is working fine.

Mit freundlichen Grüßen
i.A. Jens Teutenberg
IT Anwendungsentwickler



[Signatur-Jubiläum]

[Menke_Signatur]
Menke Agrar GmbH  | Standort Soest | Overweg 22 | 59494 Soest | Germany | www.menke-agrar.de<http://www.menke-agrar.de/>
Tel.: +49 (0) 2921/9654 -407  |  Fax: +49 (0) 2921/9654 -70407 |  jens.teutenberg@menke-agrar.de<mailto:jens.teutenberg@menke-agrar.de>

Sitz: 59494 Soest (Germany), Handelsregister: HR Arnsberg B 9696
Geschäftsführer: Dr. Andreas Menke, Dr. Wolfgang Köllner


----------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Notice: This e-mail including all attachments may contain confidential and/or privileged information. If you are not the intended
recipient (or have received this e-mail in error) please notify the sender immediately and destroy this e-mail and all attachments.
Any copying, disclosure or distribution of the material in this e-mail and the attachments without consent is strictly forbidden.
Unsolicited commercial e-mail is not permitted at this address.
Von: Michael [mailto:notifications@github.com]
Gesendet: Mittwoch, 5. Juli 2017 04:18
An: ehynds/jquery-ui-multiselect-widget
Cc: Jens Teutenberg; Author
Betreff: Re: [ehynds/jquery-ui-multiselect-widget] jQuery UI Accordion (#702)

Are you using the 2.x code and if so do you have a fiddle to demonstrate
the issue?

**Comment by mlh758** [comment]
It sounds like it's probably a stacking issue. Does anything else have the
ui-front class on it when the widget is open? I'm guessing the tab and
accordion both use it.

**Comment by jteutenberg2** [comment]
I am not the expert in jQuery UI.
It is the first time that I use it.
I only defined an accordion and a tab and placed the multiselect widget on it.

Mit freundlichen Grüßen
i.A. Jens Teutenberg
IT Anwendungsentwickler



[Signatur-Jubiläum]

[Menke_Signatur]
Menke Agrar GmbH  | Standort Soest | Overweg 22 | 59494 Soest | Germany | www.menke-agrar.de<http://www.menke-agrar.de/>
Tel.: +49 (0) 2921/9654 -407  |  Fax: +49 (0) 2921/9654 -70407 |  jens.teutenberg@menke-agrar.de<mailto:jens.teutenberg@menke-agrar.de>

Sitz: 59494 Soest (Germany), Handelsregister: HR Arnsberg B 9696
Geschäftsführer: Dr. Andreas Menke, Dr. Wolfgang Köllner


----------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Notice: This e-mail including all attachments may contain confidential and/or privileged information. If you are not the intended
recipient (or have received this e-mail in error) please notify the sender immediately and destroy this e-mail and all attachments.
Any copying, disclosure or distribution of the material in this e-mail and the attachments without consent is strictly forbidden.
Unsolicited commercial e-mail is not permitted at this address.
Von: Michael [mailto:notifications@github.com]
Gesendet: Mittwoch, 5. Juli 2017 14:15
An: ehynds/jquery-ui-multiselect-widget
Cc: Jens Teutenberg; Author
Betreff: Re: [ehynds/jquery-ui-multiselect-widget] jQuery UI Accordion (#702)

It sounds like it's probably a stacking issue. Does anything else have the
ui-front class on it when the widget is open? I'm guessing the tab and
accordion both use it.

On Wed, Jul 5, 2017, 12:13 AM jteutenberg2 <notifications@github.com<mailto:notifications@github.com>> wrote:

> I am using 2.0.1.
> I don’t have a fiddle to demonstrate it.
> The generated list box isn’t shown in the foreground.
> It is covered by the title of the accordion.
>
> [cid:image004.png@01D2F55E.26E94B40]
>
> Same problem on jQuery UI tabs.
> It isn’t possible to use the multiselect widget on a tab.
> The list is covered by the content of the tab.
> [cid:image005.png@01D2F55E.26E94B40]
>
> The problem exists with different browsers.
>
> Outside the accordion and tab it is working fine.
>
> Mit freundlichen Grüßen
> i.A. Jens Teutenberg
> IT Anwendungsentwickler
>
>
>
> [Signatur-Jubiläum]
>
> [Menke_Signatur]
> Menke Agrar GmbH | Standort Soest | Overweg 22 | 59494 Soest | Germany |
> www.menke-agrar.de<http://www.menke-agrar.de/<http://www.menke-agrar.de%3chttp:/www.menke-agrar.de/>>
> Tel.: +49 (0) 2921/9654 -407 | Fax: +49 (0) 2921/9654 -70407 |
> jens.teutenberg@menke-agrar.de<mailto:jens.teutenberg@menke-agrar.de<mailto:jens.teutenberg@menke-agrar.de%3cmailto:jens.teutenberg@menke-agrar.de>>
>
> Sitz: 59494 Soest (Germany), Handelsregister: HR Arnsberg B 9696
> Geschäftsführer: Dr. Andreas Menke, Dr. Wolfgang Köllner
>
>
> ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------
>
> Notice: This e-mail including all attachments may contain confidential
> and/or privileged information. If you are not the intended
> recipient (or have received this e-mail in error) please notify the sender
> immediately and destroy this e-mail and all attachments.
> Any copying, disclosure or distribution of the material in this e-mail and
> the attachments without consent is strictly forbidden.
> Unsolicited commercial e-mail is not permitted at this address.
> Von: Michael [mailto:notifications@github.com]
> Gesendet: Mittwoch, 5. Juli 2017 04:18
> An: ehynds/jquery-ui-multiselect-widget
> Cc: Jens Teutenberg; Author
> Betreff: Re: [ehynds/jquery-ui-multiselect-widget] jQuery UI Accordion
> (#702)
>
> Are you using the 2.x code and if so do you have a fiddle to demonstrate
> the issue?
>
> On Tue, Jul 4, 2017, 6:43 AM jteutenberg2 <notifications@github.com

**Comment by mlh758** [comment]
I'm assuming you're familiar with css though? I'm asking if the `ui-front` css class is on any other elements when the menu is open. [This is the documentation](https://api.jqueryui.com/theming/stacking-elements/) on stacking elements in jQuery UI.

Basically, open the menu and put `$('.ui-front')` in the console and see if it is in multiple places.

**Comment by jteutenberg2** [comment]
It is working now. Thanks.

Mit freundlichen Grüßen
i.A. Jens Teutenberg
IT Anwendungsentwickler



[Signatur-Jubiläum]

[Menke_Signatur]
Menke Agrar GmbH  | Standort Soest | Overweg 22 | 59494 Soest | Germany | www.menke-agrar.de<http://www.menke-agrar.de/>
Tel.: +49 (0) 2921/9654 -407  |  Fax: +49 (0) 2921/9654 -70407 |  jens.teutenberg@menke-agrar.de<mailto:jens.teutenberg@menke-agrar.de>

Sitz: 59494 Soest (Germany), Handelsregister: HR Arnsberg B 9696
Geschäftsführer: Dr. Andreas Menke, Dr. Wolfgang Köllner


----------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Notice: This e-mail including all attachments may contain confidential and/or privileged information. If you are not the intended
recipient (or have received this e-mail in error) please notify the sender immediately and destroy this e-mail and all attachments.
Any copying, disclosure or distribution of the material in this e-mail and the attachments without consent is strictly forbidden.
Unsolicited commercial e-mail is not permitted at this address.
Von: Michael [mailto:notifications@github.com]
Gesendet: Mittwoch, 5. Juli 2017 15:25
An: ehynds/jquery-ui-multiselect-widget
Cc: Jens Teutenberg; Author
Betreff: Re: [ehynds/jquery-ui-multiselect-widget] jQuery UI Accordion (#702)


I'm assuming you're familiar with css though? I'm asking if the ui-front css class is on any other elements when the menu is open. This is the documentation<https://api.jqueryui.com/theming/stacking-elements/> on stacking elements in jQuery UI.

Basically, open the menu and put $('.ui-front') in the console and see if it is in multiple places.

**Comment by mlh758** [comment]
What did you change?
